### PR TITLE
[Backport] Android platform fixes

### PR DESCRIPTION
**Pull request by MeharDT**
# PR Details
This PR backports Android platform fixes from 4.1-beta to the 4.0 branch. Please see PR's #1146 and #1018 (backport courtesy of @Basewq) for more information.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: sources/core/Stride.Core/PlatformFolders.cs [hunk 1]
```diff
@@ -182,22 +184,27 @@ private static string GetApplicationTemporaryDirectory()
...
+        private static string GetApplicationExecutableDirectory()
         {
 #if STRIDE_PLATFORM_WINDOWS_DESKTOP || STRIDE_PLATFORM_MONO_MOBILE || STRIDE_PLATFORM_UNIX
             var executableName = GetApplicationExecutablePath();
             if (!string.IsNullOrEmpty(executableName))
             {
                 return Path.GetDirectoryName(executableName);
             }
-    #if STRIDE_RUNTIME_CORECLR
+#if STRIDE_RUNTIME_CORECLR
```
Comment (xen2) [inline sources/core/Stride.Core/PlatformFolders.cs:203]: Any reason you adjusted the tab?
I don't mind much, but still they were helpful to understand the #ifdef structure.

Comment (xen2) [reply sources/core/Stride.Core/PlatformFolders.cs:203]: Anyway, not a big deal, will merge as is
4.1 looks OK for that.

Comment (MeharDT) [reply sources/core/Stride.Core/PlatformFolders.cs:203]: Odd, that file was copied over from Basewq's backport but the tabs are present in their commit.

https://github.com/Basewq/stride/blob/2e387ef78f1a4ba7211564e08249c2b9ee2f3642/sources/core/Stride.Core/PlatformFolders.cs

Might be Visual Studio weirdness, I'll put the tab back in and sneak it into the DebugText Color fix backport for 4.0.

Comment (xen2) [reply sources/core/Stride.Core/PlatformFolders.cs:203]: No need to bother, `master` is OK

**Pull request merged**